test/io_server_test.dart

```diff
@@ -16,7 +16,10 @@ import 'test_util.dart';
 void main() {
   var server;
   setUp(() async {
-    server = await IOServer.bind(InternetAddress.LOOPBACK_IP_V4, 0);
+    var address = Platform.environment.containsKey('TRAVIS')
+        ? InternetAddress.LOOPBACK_IP_V4
+        : InternetAddress.LOOPBACK_IP_V6;
```

Comment:
Can you add a comment explaining why we're testing on IPv6 on Travis and only Travis?

Reply:
Sure. The chances that the `git blame` becomes at all difficult to find before the comment becomes a lie seems incredibly low but we can add one for now.

https://github.com/dart-lang/shelf/pull/89